### PR TITLE
Improves labels/content and fieldset component added

### LINKED ISSUE #313: Improve labels and content
_AS a user,
I NEED information and form labels to be written in plain English,
SO that I can complete the process easily first-time unaided._

## Links and resources

Copy document:
https://docs.google.com/document/d/1g7ET6R6PlLQVHjgrP8dwM51vttOLnTgIQIkUm0aFsbA/edit?usp=sharing

## Acceptance Criteria

- [ ] Form labels and content to follow what is outlined in the linked document
- [ ] Works on target devices and browsers

### PR DESCRIPTION
**Pull request by ritapatel7**
@joelcarr On expenses page where I've added fieldset the inputs are not applying the grid rule. Seems like this does not work on fieldsets? What's the best way to fix this?

Closes #313.

### CODE REVIEW

**Comment by joelcarr** [comment]
@ritapatel7 `display: grid; grid-gap: 2rem;` will only work on the element's direct children so we will have to apply these rules to the `.Fieldset` class.

**Comment by ritapatel7** [comment]
@joelcarr The grid was added to fieldset class.

**Comment by joelcarr** [comment]
It seems that [it is a browser issue](https://github.com/w3c/csswg-drafts/issues/321). The best way to fix this would be to place a `<div class="Fieldset-contents">[the legend and yield stuff]</div>` wrapping the contents of the Fieldset.

File: features/step_definitions/session_review_steps.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
 end
 
 And(/^I should see a humanised indication of recording methods used$/) do
-  expect(page).to have_content('voice recording, video recording, and researcher’s written notes')
+  expect(page).to have_content('Video recording, Voice recording, and Researcher’s notes')
```
Comment (tosbourn) [inline features/step_definitions/session_review_steps.rb]: Should this section have sentence casing after commas?
